tests/app/submitter/test_submitter.py

```diff
@@ -78,3 +78,31 @@ def test_when_fail_to_publish_message_then_returns_false(self):
 
             # Then
             self.assertFalse(published, 'send_message should fail to publish message')
+
+    def test_log_submitter_send_message(self):
+        # Given
+        with patch('app.submitter.submitter.BlockingConnection'):
+            # When
+            sent_message = LogSubmitter().send_message(message={}, queue='test_queue')
+
+            # Then
+            self.assertEqual(sent_message, True)
+
+    def test_get_submitter_settings_enabled(self):
+        settings.EQ_RABBITMQ_ENABLED = True
+        rabbit_submitter = SubmitterFactory.get_submitter()
+        self.assertEqual(type(rabbit_submitter), RabbitMQSubmitter)
+
+    def test_get_submitter_settings_not_enabled(self):
+        settings.EQ_RABBITMQ_ENABLED = False
+        log_submitter = SubmitterFactory.get_submitter()
+        self.assertEqual(type(log_submitter), LogSubmitter)
+
+    def test_submitter_send_message(self):
+        sent_message_test = Submitter.send_message(self, message={}, queue='test_queue')
+        self.assertEqual(sent_message_test, False)
+
+    def test_submitter_encrypt_message(self):
+        encrypt_message_test = Submitter.encrypt_message(message={'test': 'test'})
+        self.assertIsInstance(encrypt_message_test, cls=str)
+        self.assertNotEqual(encrypt_message_test, 'test')
```

Comment:
yes that is correct. however the test isn't. this is currently testing that the entire message content is 'test' rather than the messages containing the word test